### PR TITLE
Documentation for label-based access control

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

Added documentation for new feature -> label-based access control 
The feature is the extension of access-control with clause privileges

### Pull request type
- [X] New documentation page (same page, additional documentation)

Vla: Resolved https://github.com/memgraph/docs/issues/546

### Checklist:

- [X] I checked all content with Grammarly
- [X] I performed a self-review of my code
- [ ] I made corresponding changes to the rest of the documentation
- [X] The build passes locally
- [X] My changes generate no new warnings or errors

### CODE REVIEW

**Review by BorisTasevski — APPROVED** [review]

File: docs/reference-guide/security.md [hunk 1]
```diff
@@ -166,3 +166,93 @@ executing the queries returned from a transformation, in other words, the
...
+each label (e.g. `:L1`), or `*` for specifying all labels in the graph
+- `user_or_role` is the already created user or role in Memgraph
+
+For example, granting a `READ` permission on labels `L1` and `L2` would be written as:
+
+```cypher
+GRANT READ ON LABELS :L1, :L2 TO charlie;
+```
+
+while granting both read and edit permissions for all labels in the graph, would be written as:
```
Comment (vpavicic) [inline docs/reference-guide/security.md:203]:
```suggestion
while granting both `READ` and `EDIT` permissions for all labels in the graph, would be written as:
```

File: docs/reference-guide/security.md [hunk 2]
```diff
@@ -166,3 +166,93 @@ executing the queries returned from a transformation, in other words, the
...
+
+```cypher
+GRANT UPDATE ON LABELS * TO charlie;
+```
+
+
+The same functionality could be executed with the `DENY` command, which is an inverse 
+command to `GRANT`. By using deny, users are getting a permission level one level
```
Comment (vpavicic) [inline docs/reference-guide/security.md:211]:
```suggestion
command to `GRANT`. By using `DENY`, users are getting a permission level one level 
```

File: docs/reference-guide/security.md [hunk 3]
```diff
@@ -166,3 +166,93 @@ executing the queries returned from a transformation, in other words, the
...
+Granting a certain set of edge type permissions can be done similarly to the
+clause privileges by issuing the following command:
+
+```cypher
+GRANT permission_level ON EDGE_TYPES edge_type_list TO user_or_role;
+```
+
+with the same legend as the node permissions.
+
+For example, granting a `READ` permission on edge_type `:CONNECTS` would be written as:
```
Comment (vpavicic) [inline docs/reference-guide/security.md:233]:
```suggestion
For example, granting a `READ` permission on edge type `:CONNECTS` would be written as:
```

File: docs/reference-guide/security.md [hunk 4]
```diff
@@ -166,3 +166,93 @@ executing the queries returned from a transformation, in other words, the
...
+```
+
+### Revoking label-based permissions
+To revoke any of the label-based permissions, users can use one of the following commands:
+
+```cypher
+REVOKE (LABELS | EDGE_TYPES) label_or_edge_type_list FROM user_or_role
+```
+
+with the legend:
```
Comment (vpavicic) [inline docs/reference-guide/security.md:246]:
```suggestion
where:
```

File: docs/reference-guide/security.md [hunk 5]
```diff
@@ -166,3 +166,93 @@ executing the queries returned from a transformation, in other words, the
...
+- `label_or_edge_type_list` is a list of labels or edge types with a colon in front of each
+label or edge type (or `*` for specifying all labels or edge types)
+- `user_or_role` is the existing user or role in Memgraph
+
+### Show privileges for label-based access control
+For checking the privileges the existing user or role has in Memgraph, it is enough to write
```
Comment (vpavicic) [inline docs/reference-guide/security.md:252]:
```suggestion
To check which privileges an existing user or role has in Memgraph, it is enough to write
```

File: docs/reference-guide/security.md [hunk 6]
```diff
@@ -166,3 +166,93 @@ executing the queries returned from a transformation, in other words, the
...
+
+```cypher
+SHOW PRIVILEGES FOR user_or_role;
+```
+
+and all the values of clause privileges as well as label-based permissions will be displayed.
```
Comment (vpavicic) [inline docs/reference-guide/security.md:258]:
```suggestion
and all the values of clause privileges, as well as label-based permissions, will be displayed.
```

**Comment by vpavicic** [comment]
Please define where you want to merge this branch
[master<YOUR_BRANCH]

**Comment by Josipmrden** [comment]
@vpavicic  fixed pr comments and added 2 sentences with the permission level NOTHING

**Pull request merged**